src/rdkafka_broker.c

```diff
@@ -3661,6 +3720,7 @@ rd_kafka_fetch_reply_handle (rd_kafka_broker_t *rkb,
 					   RD_KAFKAP_STR_PR(&topic),
 					   hdr.Partition);
 				rd_kafka_buf_skip(rkbuf, hdr.MessageSetSize);
+                                rd_kafka_abrt_state_destroy(abrt);
```

Comment:
I put the NULL in the rd_kafka_abrt_state_destroy method since it avoids a lot of repetition. i'll move that out as you're assuming it's working. that way alternatively i could change the name of the method.